sys/shell/commands/sc_gnrc_rpl.c

```diff
@@ -283,18 +301,35 @@ int _gnrc_rpl(int argc, char **argv)
             return _gnrc_rpl_operation(false, argv[2]);
         }
     }
+    else if (strcmp(argv[1], "set") == 0) {
+        if (argc > 2) {
+#ifndef GNRC_RPL_WITHOUT_PIO
+            if (strcmp(argv[2], "pio") == 0) {
+                if ((argc == 5) && (strcmp(argv[3], "on") == 0)) {
+                    return _gnrc_rpl_set_pio(argv[4], true);
+                }
+                else if ((argc == 5) && (strcmp(argv[3], "off") == 0)) {
+                    return _gnrc_rpl_set_pio(argv[4], false);
+                }
```

Comment:
sounds interesting. I can extend the output of `rpl` in a follow-up to show the PIO status for all dodags/instances